cobalt/BUILD.gn

```diff
@@ -79,10 +81,8 @@ group("default") {
         "//components/crx_file",
         "//components/prefs",
         "//components/update_client",
+        "//third_party/llvm-project/libunwind:unwind_evergreen",
```

Comment:
I know this isn't directly part of this PR, but this if block should not be nested in this `if (sb_is_modular)` block, it should be its own conditional (for cleanliness)

Reply:
i think you are maybe fighting the agreement that evergreen builds are modular.
I think this should remain here, otherwise it looks out of place

Reply:
@y4vor  thoughts ?